vignettes/xcmsPreprocess.Rnw

```diff
@@ -237,6 +237,10 @@ precise peak integrals.  The peak attributes \Robject{sn}
 (Signal/Noise Ratio) and \Robject{egauss} (root-mean-square-error of
 the gaussian fit) can be used to assess the peak quality.
 
+\textbf{findPeaks.centWaveWithPredictedIsotopeROIs} \\
+The method \Rmethod{findPeaks.centWaveWithPredictedIsotopeROIs} performs two steps of peak picking.
+In the first peak picking step, the method \Rmethod{findPeaks.centWave} is used as described in the previous section. In the first peak picking step, the method \Rmethod{findPeaks.centWave} is used on basis of a set of predicted regions of interest (ROIs) which are specific for isotope peaks. This enables a more sensitive detection of isotope peaks.
```

Comment:
This description could be extended a little more. Also, you should provide a help (Rd) page for your method.